test/libsolidity/syntaxTests/inheritance/override/triangle.sol

```diff
@@ -0,0 +1,5 @@
+contract A { function f() public pure virtual {} }
+contract B is A { function f() public pure virtual override {} }
```

Comment:
Not `virtual` by default if `override`? ;)

Reply:
Nope, ``virtual`` is only ever the default in interfaces :-).

Reply:
Otherwise we'd also need ``final`` and people would forget to use it, etc., etc. ...